sat/cli/jobstat/main.py

```diff
@@ -54,7 +55,18 @@ def do_jobstat(args):
             display_headings=args.fields,
             print_format=args.format,
         )
+        wrapper = textwrap.TextWrapper(width=20)
         for row in application_data:
+            if args.format == 'pretty':
```

Comment:
I would slightly prefer to have this formatting code refactored into a helper function. Running SonarLint against this file reports that the `do_jobstat` function has a cognitive complexity of 18 which is slightly higher than the recommended maximum of 15.

Cognitive complexity is basically a measure of the complexity of a function, i.e. number of "if" statements and possible branches of the code. https://www.sonarsource.com/resources/cognitive-complexity/

https://gist.github.hpe.com/eli-kamin/c4b29404161ab0b27a881c5bd86649a5

Reply:
Marking this conversation as resolved since the refactor was done.